README.md

```diff
@@ -13,36 +13,52 @@ Due to a popular python library called scikit-learn requires bzip2 library which
 I have also built maxjiang/rpi-jupyter:datascience (based on 1.1) so that you have most of the data science packages installed without installing/compiling them yourself.
 
 ### Installing
-Go to [Hypriot OS](http://blog.hypriot.com/) and follow the steps to get the Raspberry Pi docker ready. Then, run the following:
+Go to [Hypriot OS](https://blog.hypriot.com/downloads) and follow the steps to get this OS. However if you have [Raspbian Jessie](https://www.raspberrypi.org/downloads/raspbian/) OS installed, you are totally fine too, it will work. 
```

Comment:
I actually did not realise Docker officially supported Raspberry Pi until now. Mentioning Hypriot OS is not necessary here so please rephrase if you like.